### ISSUE TITLE
Skeleton must be multi-threaded

### ISSUE THREAD

**Issue by yegor256** [yegor256/jpeek#177]
Let's run it in parallel threads.

**Comment by 0crat** [comment]
Job `gh:yegor256/jpeek#177` is not assigned, can't get start time

**Comment by 0crat** [comment]
Job [#177](https://github.com/yegor256/jpeek/issues/177) was not in scope
